staging/src/k8s.io/pod-security-admission/policy/check_hostNamespaces.go

```diff
@@ -25,15 +25,18 @@ import (
 )
 
 /*
-Sharing the host namespaces must be disallowed.
+Sharing the host namespaces must be disallowed.  The only exception is the user namespace as it is optional for now.
 
 **Restricted Fields:**
 
 spec.hostNetwork
 spec.hostPID
 spec.hostIPC
 
-**Allowed Values:** undefined, false
+**Allowed Values:**
```

Comment:
revert this change, the existing allowed values were correct

Reply:
Fixed, thanks